lib/server/db/postgres/postgres_test.go

```diff
@@ -4,13 +4,12 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package postgres_test
+package postgres
```

Comment:
If we're using a mock, why do we need to access the unexported `exists` function? We can verify the interaction and parameters by calling `CreateDatabase`.